src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,25 @@ const createCustomerValidator = () => {
     });
 };
 
+const getCustomersValidator = () => {
+    return celebrate({
+        [Segments.QUERY]: Joi.object().keys({
+            offset: Joi.string(),
```

Comment:
Poderia já receber o campo `offset` como sendo um número inteiro, e com um valor padrão caso não fosse informado.
```suggestion
            offset: Joi
                .number()
                .integer()
                .min(0)
                .default(0),
```